### PR TITLE
Add statistics

### PR DESCRIPTION
**Pull request by mredigonda**

### CODE REVIEW

File: client/src/app/admin/panel/dashboard/admin-panel-stats.js [hunk 1]
```diff
@@ -1,25 +1,233 @@
...
+import API from 'lib-app/api-call';
 
 class AdminPanelStats extends React.Component {
 
+    state = {
+        loading: true,
+        rawForm: {
+            dateRange: {
+                startDate: 202009130000,
+                endDate: 202009132359
```
Comment (mredigonda) [inline client/src/app/admin/panel/dashboard/admin-panel-stats.js:21]: I need to provide better defaults.

File: server/controllers/system/stats.php [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+                ],
+                'owners' => [
+                    'validation' => DataValidator::oneOf(DataValidator::validOwnersId(), DataValidator::nullType()),
+                    'error' => ERRORS::INVALID_OWNER_FILTER
+                ],
+            ]
+        ];
+    }
+
+    public function handler() {
```
Comment (ivandiazwm) [inline server/controllers/system/stats.php:68]: @mredigonda please escape variables anyways before adding it them to the query

Comment (ivandiazwm) [reply server/controllers/system/stats.php:68]: @mredigonda also, keep in mind you can use bindings with redbean
```
RedBean::getCell($query, $bindings);
```

**Review by ivandiazwm — CHANGES_REQUESTED** [review]
lgtm but use bindings before inserting variables in the sql query strings

**Comment by mredigonda** [comment]
🥳 Tell me if something breaks. It would be amazing if we could test the SQL performance of this.

**Pull request merged**